Anature/src/application/pools/MovePool.java

```diff
@@ -107,6 +125,186 @@ private static void generateMoves()
 						.withAccuracy(85)
 						.create());
 		
+		mMoves.put(MoveIds.Cinder,
+				new MoveBuilder<Cinder>(MoveIds.Cinder).withName("Cinder")
+						.withType(Type.Fire)
+						.doesDamage(true)
+						.isPhysicalAttack(true)
+						.withTotalMovePoints(20)
+						.withMovePower(40)
+						.withAccuracy(100)
+						.create());
+		
+		mMoves.put(MoveIds.Smash,
+				new MoveBuilder<Smash>(MoveIds.Smash).withName("Smash")
+						.withType(Type.Fire)
+						.doesDamage(true)
+						.isPhysicalAttack(true)
+						.withTotalMovePoints(25)
+						.withMovePower(25)
+						.withAccuracy(100)
+						.create());
+		
+		mMoves.put(MoveIds.Fire_Torture,
+				new MoveBuilder<FireTorture>(MoveIds.Fire_Torture).withName("Fire Torture")
+						.withType(Type.Fire)
+						.doesDamage(true)
+						.isPhysicalAttack(true)
+						.withTotalMovePoints(20)
+						.withMovePower(90)
+						.withAccuracy(80)
+						.create());
+		
+		mMoves.put(MoveIds.Smog_Wave,
+				new MoveBuilder<SmogWave>(MoveIds.Smog_Wave).withName("Smog Wave")
+						.withType(Type.Normal)
+						.doesDamage(false)
```

Comment:
Smog Wave is a normal type? Is that correct, cause it sounds like a poison type move.